spec/controllers/users_controller_spec.rb

```diff
@@ -79,30 +84,41 @@
     end
 
     context 'sending email' do
-      before { ActionMailer::Base.deliveries.clear }
+      let(:stripe_helper) { StripeMock.create_test_helper }
+      after { StripeMock.stop }
+
+      before do
+        StripeMock.start
+        ActionMailer::Base.deliveries.clear
+      end
 
       it 'sends out email to the user with valid inputs' do
         user_attributes = Fabricate.attributes_for(:user)
-        post :create, user: user_attributes
+        post :create, user: user_attributes, stripToken: stripe_helper.generate_card_token
```

Comment:
I remove it and still pass, I guess it's because I am using strip-ruby-mock. I'll check it out later.